persistence/sql/persister_session.go

```diff
@@ -27,3 +28,14 @@ func (p *Persister) CreateSession(ctx context.Context, s *session.Session) error
 func (p *Persister) DeleteSession(ctx context.Context, sid uuid.UUID) error {
 	return p.c.Destroy(&session.Session{ID: sid}) // This must not be eager or identities will be created / updated
 }
+
+func (p *Persister) DeleteSessionsFor(ctx context.Context, sid uuid.UUID) error {
+	count, err := p.c.RawQuery("DELETE FROM sessions WHERE identity_id =?", sid).ExecWithCount()
```

Comment:
We don't need `ExecWithCount`, `Exec` should be enough

Reply:
done